client/src/components/Attachment/UploadAttachment.js

```diff
@@ -45,6 +45,7 @@ const UploadAttachment = ({
     const selectedAttachment = new Attachment({
       fileName: file.name,
       mimeType: file.type,
+      caption: file.name.substr(0, file.name.lastIndexOf(".")),
```

Comment:
This fails if for some reason the filename doesn't have a dot in it (in which case you should just take the whole filename).